config/intel-warnings/oneapi/developer-general

```diff
@@ -0,0 +1,11 @@
+-Wreorder
+-Wstrict-aliasing
+-Wreorder
+# -Winline warnings aren't included here because, for at least
+# GNU compilers, this flag appears to conflict specifically with
+# the -Og optimization level flag added for Debug and Developer
+# builds and will produce warnings about functions not being
+# considered for inlining. The flag will be added to the list
+# of compiler flags separately if developer warnings are enabled
+# and the build type is not Debug or Developer
+#-Winline
```

Comment:
Can probably remove this comment and the commented out flag since it's mostly for GNU